pkgs/development/web/nodejs/v20.nix

```
@@ -44,5 +44,10 @@ buildNodejs {
       stripLen = 1;
       hash = "sha256-6y3aEqxNC4iTQEv1oewodJrhOHxjp5xZMq1P1QL94Rg=";
     })
+    # fixes test failure, remove once included in release
+    (fetchpatch {
+      url = "https://github.com/nodejs/node/commit/b6fe731c55eb4cb9d14042a23e5002ed39b7c8b7.patch";
```

```suggestion
      url = "https://github.com/nodejs/node/commit/b6fe731c55eb4cb9d14042a23e5002ed39b7c8b7.patch?full_index=1";
```
We need the `?full_index=1` so the patch is stable (otherwise it might change as GitHub updates the length of the index as the repo grows)

---

Not with `fetchpatch` (rather than `fetchpatch2`), as it turns out. You get to choose between stripping `index` lines properly and handling renames…